packages/common/src/types/question.ts

```diff
@@ -17,6 +17,7 @@ export interface Question {
   lastUpdate: string
   links: ResourceLink[]
   lastDateToReply?: string
+  contactInfo: string[]
```

Comment:
This is used as `question.contactInfo &&` suggesting that this could be undefined or null. Please update interface to reflect that. `contactInfo?: string[]` or `contactInfo: string[] | null`